src/components/date-picker/pickers/Picker.ts

```diff
@@ -101,38 +88,8 @@ export abstract class Picker {
         });
     }
 
-    private handleCloseForFlatpickr(selectedDates) {
-        return new Promise((resolve) => {
-            // Since we allow manual editing of the input value, and
-            // flatpickr only picks up these changes when the user presses
-            // enter in the input, we need to check if the input string
-            // and the underlying value match.
-            //
-            // If this timeout is set to 0, we get a race-condition where
-            // the value is sometimes updated from the input-string, and
-            // sometimes not.
-            const timeout = 100;
-            setTimeout(() => {
-                const momentInputDate = moment(
-                    this.flatpickr.input.value,
-                    this.dateFormat,
-                    this.getMomentLang()
-                );
-                let pickerDate = this.getPickerDate(selectedDates);
-                const isSameInput = momentInputDate.isSame(pickerDate);
-                if (!isSameInput) {
-                    if (momentInputDate.isValid()) {
-                        pickerDate = momentInputDate.toDate();
-                        this.flatpickr.setDate(pickerDate);
-                    } else {
-                        pickerDate = null;
-                        this.flatpickr.clear();
-                    }
-                }
-                this.change.emit(pickerDate);
-                resolve(pickerDate);
-            }, timeout);
-        });
+    protected getMomentLang() {
+        return this.language === 'no' ? 'nb' : this.language;
```

Comment:
With the new DateFormatter, are we using this `language` property anywhere?

Reply:
Bump

Reply:
We are using the `getMomentLang` I guess?